01-front-end-basics/templating-handlebars/27-28/readme.md

```diff
@@ -19,110 +21,91 @@
+## Explore Handlebars
 
-    <!-- custom script -->
-    <script type="text/javascript" src="js/app.js"></script>
-  </head>
-  ```
+We'll use the [HTML strings lab](https://github.com/SF-WDI-LABS/html_strings.git) as our starting point. Check out the `handelbars` branch. 
 
-2. Next create an element in your `index.html` where you will append the data from your template. Give this element an `id` (e.g. `results`) so you can select it very specifically with jQuery.
+### What is happening in `index.html`?
 
-  ```html
-  <body>
-    <div class="container">
-      <div class="row">
-        <div class="col-md-6 col-md-offset-3">
-          <div id="results">
-          </div>
-        </div>
-      </div>
-    </div>
-  </body>
-  ```
+**Create a mold**
 
-3. Create the template inside your `results` element. Give your template an `id` (e.g. `tracks-template`) so you can select it with jQuery. This template acts as an HTML "mad lib" which you'll fill with data. The `{{}}` parameters will be replaced by the data that comes back from your API call.
-
-  ```html
-  <body>
-    <div class="container">
-      <div class="row">
-        <div class="col-md-6 col-md-offset-3">
-          <div id="results">
-
-            <!-- handlebars template -->
-            <script id="tracks-template" type="text/x-handlebars-template">
-              {{#each tracks}}
-                <p><strong>{{name}}</strong> by {{artists.[0].name}}</p>
-              {{/each}}
-            </script>
-
-          </div>
-        </div>
-      </div>
-    </div>
-  </body>
-  ```
+1) We make a `script` tag with an `id` that describes what the template will be for.
 
-  **Note:** The example above uses the Spotify API. We use `{{each}}` to iterate through the tracks that come back from Spotify. Each track has a `name` and an array of `artists`. Notice the extra `.` we need when accessing a value from an array (this syntax is specific to Handlebars). `artists.[0]` gives us the first artist from the array.
+2) Using Handlebars, we create a structure we'd like to use for part of the page. Ours is for each student's link. This structure should be inside the script tag. Note the use of `{{ }}` to identify where data will go. 
 
-4. Compile your template in `js/app.js`. Handlebars has a function called `Handlebars.compile(source)`. The source is takes in is the html from inside your template script. Handlebars' `compile` function returns a function, which we'll save to a variable called `template`. This isn't a special varible name; it just reminds us what the function does. Later, we'll use the `template` function to  pass in data to render in the template part of the page.
+3) We make sure the data format we use matches what we have available to us. The example above uses the Spotify API. In this example, each student has a `first_name`, a `last_name`, and a `github_username`. 
```

Comment:
> We make sure the data format we use matches what we have available to us. The example above uses the Spotify API. In this example, each student has a `first_name`, a `last_name`, and a `github_username`. 

suggesting change:
We make sure that the data format we use matches the format of the data that is available to us. The example above uses the Spotify API. In this example, each student has a `first_name`, a `last_name`, and a `github_username`. 

Also I'm confused...it says Spotify api and then says `first_name` `last_name` etc... those don't seem related at first glance.